hoodie-client/src/test/java/com/uber/hoodie/TestHoodieClientOnCopyOnWriteStorage.java

```diff
@@ -178,6 +179,34 @@ public void testFilterExist() throws Exception {
     assertTrue(result.size() == 25);
   }
 
+  @Test
+  public void testTagLocation() throws Exception {
```

Comment:
Not sure if we need this test (in the interest of keeping this file from growing more).. This just tests index.tagLocation() which should be tested already..

Reply:
This test specifically test tagLocation.  You mean to remove this test entirely?

Reply:
I am asking how this is different from something that tests index.tagLocation.. if not, we can remove the test and make other parts of current tests call the upper level API (readClient.tagLocation) instead of index.tagLocation?  This way you get test coverage for the API without adding more redundant tests.?